From 59a61be1b7b7cb45f5494b980b9f37e2838d6e6a Mon Sep 17 00:00:00 2001
From: Astro <astro@spaceboyz.net>
Date: Wed, 10 Jun 2020 19:29:25 +0200
Subject: [PATCH] limit vivado threads to 1

---
 migen/build/xilinx/vivado.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/migen/build/xilinx/vivado.py b/migen/build/xilinx/vivado.py
index d2cf101..e9e4b77 100644
--- a/migen/build/xilinx/vivado.py
+++ b/migen/build/xilinx/vivado.py
@@ -149,6 +149,7 @@ class XilinxVivadoToolchain:
 
         # script that calls above steps
         tcl = []
+        tcl.append("set_param general.maxThreads 1")
         tcl.append("source \"{}_route.tcl\"".format(build_name))
         tcl.append("source \"{}_bitstream.tcl\"".format(build_name))
         tools.write_to_file(build_name + ".tcl", "\n".join(tcl))
-- 
2.26.2

