diff --git a/bridge/mattermost/helpers.go b/bridge/mattermost/helpers.go
index 14b7469d..d9b77bdf 100644
--- a/bridge/mattermost/helpers.go
+++ b/bridge/mattermost/helpers.go
@@ -206,6 +206,10 @@ func (b *Bmattermost) skipMessage(message *matterclient.Message) bool {
 		return true
 	}
 
+	if message.Username == "github" {
+		return true
+	}
+
 	// if the message has reactions don't repost it (for now, until we can correlate reaction with message)
 	if message.Post.HasReactions {
 		return true
